### PR TITLE
Mergeback 1.4.2rc2

### PR DESCRIPTION
**Pull request by yunchu**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,11 +20,9 @@ All notable changes to this project will be documented in this file.
...
 ### Bug fixes
 
-- Fix F1 auto-threshold to choose best largest confidence (<https://github.com/openvinotoolkit/training_extensions/pull/2371>)
-
-### Known issues
-
-- OpenVINO(==2023.0) IR inference is not working well on 2-stage models (e.g. Mask-RCNN) exported from torch==1.13.1
+- Add workaround for the incorrect meta info M-RCNN (used for XAI) (<https://github.com/openvinotoolkit/training_extensions/pull/2437>)
+- Fix label list order for h-label classification (<https://github.com/openvinotoolkit/training_extensions/pull/2440>)
+- Modified fq numbers for lite HRNET e2e tests (<https://github.com/openvinotoolkit/training_extensions/pull/2445>)
```
Comment (goodsong81) [inline CHANGELOG.md:32]: Need to create 1.4.2 section.

File: tests/e2e/cli/semantic_segmentation/reference/Custom_Semantic_Segmentation_Lite-HRNet-18-mod2_OCR/compressed_model.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 TestToolsMPASegmentation:
   nncf:
     number_of_fakequantizers: 586
-  ptq:
-    number_of_fakequantizers: 494
+  pot:
+    number_of_fakequantizers: 15
```
Comment (goodsong81) [inline tests/e2e/cli/semantic_segmentation/reference/Custom_Semantic_Segmentation_Lite-HRNet-18-mod2_OCR/compressed_model.yml:5]: Is this intended?

**Review by goodsong81 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (yunchu) [reply CHANGELOG.md:32]: updated

**Comment by yunchu** [comment]
for the convenient collaboration, I'll merge it to the new branch in this repo then I will open a PR again for merging develop.

**Comment by yunchu** [comment]
sorry guys. I will re-open the merge-back PR again.

**Pull request merged**